setup.py

```diff
@@ -201,7 +201,6 @@ def run(self):
       description=description,
       long_description=long_description,
       author='KotlinIsland',
-      url='https://www.github.com/KotlinIsland/basedmypy',
```

Comment:
why was this removed

Reply:
![image](https://user-images.githubusercontent.com/65446343/156706810-9761a626-deb7-4a0f-a45a-6d6b0d73ad23.png)
It's linked as the "Homepage", which is more misleading imo.